### PR TITLE
[DLP] Implemented dlp_inspect_data_to_hybrid_job_trigger and tested using mocking approach

### PR DESCRIPTION
**Pull request by Avani-Thakker-Crest**
## Description
Implemented dlp_inspect_data_to_hybrid_job_trigger and tested using **mocking approac**h. JSON equivalent: https://cloud.google.com/dlp/docs/how-to-hybrid-jobs#formatting

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: dlp/snippets/inspect_content.py [hunk 1]
```diff
@@ -1320,6 +1320,94 @@ def callback(message):
 # [END dlp_inspect_gcs_with_sampling]
 
 
+# [START dlp_inspect_data_to_hybrid_job_trigger]
```
Comment (annadata) [inline dlp/snippets/inspect_content.py:1323]: dlp_inspect_data_to_hybrid_job_trigger
If we are introducing new region tag, please use:
dlp_inspect_send_data_to_hybrid_job_trigger

**Review by annadata — APPROVED** [review]

File: dlp/snippets/inspect_content.py (hunk 1 above)
Comment (Avani-Thakker-Crest) [reply dlp/snippets/inspect_content.py:1323]: Yes, this is part of new region tags, for java this code is already merged with dlp_inspect_data_to_hybrid_job_trigger (Link: https://github.com/GoogleCloudPlatform/java-docs-samples/pull/8013 ), so everyone of us has to update the region tag, correct. Please confirm, thank you!

Comment (annadata) [reply dlp/snippets/inspect_content.py:1323]: Let's update Java as well. Let's use dlp_inspect_send_data_to_hybrid_job_trigger

Comment (Avani-Thakker-Crest) [reply dlp/snippets/inspect_content.py]: Got it, have updated the region tag and have spread the word in the team. Thank you. Resolving conversation

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**